src/Service/Bulk/BulkUpdateUsersAssignmentsStateService.php

```diff
@@ -129,14 +128,14 @@ private function processOperation(BulkOperation $operation, BulkResult $result):
     /**
      * @throws LogicException
      */
-    private function validateStateTransition(BulkOperation $operation): void
+    private function validateStateAttribute(BulkOperation $operation): void
     {
-        if ($operation->getAttribute('state') !== Assignment::STATE_CANCELLED) {
+        if ($operation->getAttribute('state') !== Assignment::STATUS_CANCELLED) {
```

Comment:
can we change it to `$operation->getAttribute('status')` as well?

Reply:
We can, I am going to create a PR in lambda assignment manager as well to match expectations. Fixed